script/upload_to_rpm_repo

```diff
@@ -48,36 +53,17 @@ echo "%_signature gpg
 
 rm -rf $TMP_REPO
 mkdir -pv $TMP_REPO/x86_64
-
-# if .sig files exist remove them
-rm -rf dist/*.sig
-
-# Check whether each rpm is signed
-for i in $(ls dist/*.rpm); do
-    if ! rpm -v -K $i | grep "key ID ${GPG_KEY_SHORT_ID,,}: OK"; then
-        if [ "${CHANNEL}" = "release" ]; then
-            echo "No signature found, running \"./rpm-resign.exp $GPG_KEY_ID $i (REDACTED GPG_PASSPHRASE)\""
-            ./rpm-resign.exp $GPG_KEY_ID $i $GPG_PASSPHRASE
```

Comment:
If we're doing away with `rpm-resign.exp`(good stuff by the way using the `rpmsign --key-id` argument), we should also remove that script.

Reply:
Resolved by deleting the file